mobile/salto/src/commands.ts

```diff
@@ -7,10 +7,40 @@ export async function getNodesBalance() {
      let res =  await invoke("get_nodes_balance")
        .then((message) => message as NodeData[] )
        .catch((error) => console.error(error));
+      return res;
+  }
 
+  export async function getPrices(currencies: string[], assets?: string[]) {
+     let res =  await invoke("get_prices", {currencies, assets})
+       .then((message) => message)
+       .catch((error) => console.error(error));
+      return res;
+  }
+
+  export async function getCurrencies() {
+    let res =  await invoke("get_currencies")
+       .then((message) => message as string[])
+       .catch((error) => console.error(error));
       return res;
   }
 
+  export async function priceProviderAddAssets( assets: string[]) {
+    let res = await invoke("price_provider_add_assets", { newAssets: assets }).then((message) => message).catch((error) => console.error(error));
+    return res;
+  }
+
+  export async function priceProviderAddCurrencies(currencies: string[]) {
+    let res = await invoke("price_provider_add_currencies", { newCurrencies: currencies }).then((message) => message).catch((error) => console.error(error));
+    return res;
+  }
```

Comment:
this is true for both those commands.
You are doing front -> back -> front -> back -> front.
It should be front (user action) -> back (program reaction) -> front (display update)
And never more